pipe_segment/pipeline.py

```diff
@@ -70,8 +72,11 @@ def segmenter_params(self):
         return ujson.loads(self.options.segmenter_params)
 
     @property
-    def source_tables(self):
-        return self.options.source.split(",")
+    def source_tables(self) -> List[BigQueryMessagesSource]:
+        result=[]
+        for i, source in enumerate(self.options.source.split(",")):
+            result.append(BigQueryMessagesSource(table_id=source))
+        return result
 
```

Comment:
This can be `[BigQueryMessageSource(table_id=source) for source in self.options.source.split(",")]` if you want to avoid stateful computationi.

Reply:
Sure. Fixed